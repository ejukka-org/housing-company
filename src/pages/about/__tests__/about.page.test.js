import React from 'react';
import AboutPage from '../about.page';
import renderer from 'react-test-renderer';

test('Renders About page', () => {
    const component = renderer.create(
        <AboutPage />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
