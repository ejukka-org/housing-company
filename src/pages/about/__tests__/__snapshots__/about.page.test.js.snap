// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders About page 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm"
    />
    <div
      className="col-sm"
    />
    <div
      className="col-sm"
    />
  </div>
  <div
    className="row"
  >
    <div
      className="col-sm"
    />
    <div
      className="col-sm"
    />
    <div
      className="col-sm"
    />
  </div>
  <br />
  <div
    className="list-group"
    onKeyDown={[Function]}
  >
    <div
      className="list-group-item"
      data-rb-event-key={null}
      disabled={false}
      onClick={[Function]}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      />
    </div>
    <div
      className="list-group-item"
      data-rb-event-key={null}
      disabled={false}
      onClick={[Function]}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      />
    </div>
    <div
      className="list-group-item"
      data-rb-event-key={null}
      disabled={false}
      onClick={[Function]}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      />
    </div>
    <div
      className="list-group-item"
      data-rb-event-key={null}
      disabled={false}
      onClick={[Function]}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      />
    </div>
    <div
      className="list-group-item"
      data-rb-event-key={null}
      disabled={false}
      onClick={[Function]}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      />
    </div>
  </div>
  <div>
    <div
      style={
        Object {
          "height": "500px",
          "width": "500px",
        }
      }
    >
      <div
        style={
          Object {
            "height": "100%",
          }
        }
      />
      <div />
    </div>
  </div>
</div>
`;
